node/src/Commands.ts

```diff
@@ -1525,3 +1525,43 @@ export function createObjectIdletime(key: string): command_request.Command {
 export function createObjectRefcount(key: string): command_request.Command {
     return createCommand(RequestType.ObjectRefCount, [key]);
 }
+
+export type LPosOptions = {
```

Comment:
(if we decide to keep this class): when we create new classes/enums in node, we have to add them to node/npm/glide/index.ts (two places in this file), similarly to when we add them to init.py in python